lib/node_modules/@stdlib/math/base/special/asec/lib/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/**
+* Compute the inverse (arc) secant of a number.
+*
+* @module @stdlib/math/base/special/asec
+*
+* @example
+* var asec = require( '@stdlib/math/base/special/asec' );
+*
+* var v = asec( 1.0 );
+* // returns 0.0
+*
+* v = asec( 2 );
```

Comment:
Missing decimal.

Reply:
@steff456 This is undoubtedly not obvious, but, for the most part, when an input value is a floating-point number, we always include decimals, similar to Python. When a function only takes integer values, we omit the decimal to indicate that only integer inputs are to be provided.

Reply:
I think there's some other packages that need to be updated for consistency, I copied from `asech` and it is missing decimal places also 😉

Reply:
Feel free to flag those instances so we can fix them. :)